packages/backend/tsconfig.test.json

```diff
@@ -7,7 +7,10 @@
     "moduleResolution": "Node",
     "noImplicitAny": false,
     "outDir": "./tests/dist",
-    "target": "ES2020"
+    "target": "ES2020",
+    "paths": {
+      "@clerk/shared/*": ["../shared/dist/*.js"]
```

Comment:
Switching `moduleResolution` to `NodeNext` here made the tests break, so I'm using `paths` here to map out the subpath imports ([valid workaround](https://github.com/microsoft/TypeScript/issues/44848#issuecomment-987953500))